lib/cached_booking_locations_api.rb

```diff
@@ -1,11 +1,11 @@
-class CachedBookingLocationsApi < BookingLocations::Api
+class CachedBookingLocationsApi
   def initialize(cache: Rails.cache)
     @cache = cache
   end
 
   def find(location_id)
     cache.fetch(location_id, expires_in: 2.hours) do
```

Comment:
It's a uuid, there's no chance of it clashing and besides we're not caching anything else right now.